README.md

```diff
@@ -20,17 +20,17 @@ Reduce food waste by searching our database for an ingredient that you have gett
 
 - Our team frequently employed pair programming to teach each other Sass and work on Javascript together
 ### Focus on Accessibility:
-- 100% Accessibility Rating thru [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/)!
+- 100% Accessibility Rating through [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/)!
 - All colors for design checked with [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/) and passed WCAG AAA ratings
 - Fonts no smaller than 16px, larger readable recipe cards
 ![Large readable recipe cards](img/readme/readable-recipe-cards.png)
-- All buttons inputs and icons are labeled for screen readers
-- SVGs are used instead of fonticons to improve access for people who have adjusted their fonts in the browser
+- All button inputs and icons are labeled for screen readers
+- SVGs are used instead of functions to improve access for people who have adjusted their fonts in the browser
```

Comment:
This is meant to say SVGs are used instead of FONTICONS not functions lol. SVGs are vector images and fonticons are stylized text that is not as accessible for people who choose a special font for their browser